using_images/xpaas_images/decision_server.adoc

```diff
@@ -0,0 +1,137 @@
+* Execution of business processes are not supported.
+* Authoring of business rules are not supported.
+
+[[Managing-OpenShift-Decision-Server-xPaaS-Images]]
+=== Managing OpenShift Decision Server xPaaS Images
+
+As the Decision Server image is built off the OpenShift JBoss EAP xPaaS image the JBoss EAP Management CLI
+is accessible from within the container for troubleshooting purposes.  First open a
+remote shell session to the running pod:
+
+----
+$ oc rsh <pod_name>
+----
+
+Then run the following from the remote shell session to launch the JBoss EAP
+Management CLI:
+
+----
+$ /opt/eap/bin/jboss-cli.sh
+----
+
+[WARNING]
+Any configuration changes made using the JBoss EAP Management CLI on a running container will be lost when the container restarts.
+
+link:#Making-Configuration-Changes-EAP[Making configuration changes to the
+JBoss EAP instance inside the JBoss EAP xPaaS image] is different from the process you may be used to for a regular release of JBoss EAP.
+
+== Installing the Decision Server xPaaS Image Streams and Application Templates
+
+To use the Red Hat xPaaS middleware images in your OpenShift project, you must
+first
+link:../../install_config/install/first_steps.html#creating-image-streams-for-xpaas-middleware-images[install
+the image streams] and
+link:../../install_config/install/first_steps.html#creating-instantapp-templates[Source-to-Image
+(S2I) application templates].
+
+
+[[Making-Configuration-Changes-Decision-Server]]
+== Running and Configuring the Decision Server xPaaS Image
+
+You can make changes to the Decision Server configuration in the xPaaS image using either the S2I templates, or by using a modified Decision Server image.
+
+=== Using the Decision Server xPaaS Image Source-to-Image (S2I) Process
+
+The recommended method to run and configure the OpenShift Decision Server xPaaS image is to use the OpenShift S2I process together with the application template parameters and environment variables.
+
+The S2I process for the Decision Server xPaaS image works as follows:
+
+. If there is a *_pom.xml_* file in the source repository, a Maven build is triggered with the contents of `*$MAVEN_ARGS*` environment variable.
++
+By default the `package` goal is used with the `openshift` profile, including the system properties for skipping tests (`*-DskipTests*`) and enabling the Red Hat GA repository (`*-Dcom.redhat.xpaas.repo.redhatga*`).
++
+The results of a successful Maven build are installed into the local Maven repository, *_/home/jboss/.m2/repository/_*, along with all dependencies if the image is configured for offline-mode.
++
+Once the artifacts have been installed into the local Maven repository the resulting kjars are placed into *_EAP_HOME/standalone/deployments_* to deploy the rule. This includes all JAR, WAR, and EAR files from the directory within the source repository specified by `*$ARTIFACT_DIR*` environment variable. The default value of `*$ARTIFACT_DIR*` is the *_target_* directory.
```

Comment:
@huffmanca I'm not sure what the "target" directory is? Is this a directory that should be spelled out more? IE, the full pathname? Or is it a paramter that will need the directory set? If that's the case, in which file is the parameter?